modal/runner/shared/download.py

```diff
@@ -53,7 +58,7 @@ def download_model(model_name: str):
         ignore_patterns.append("*.bin")
 
     # Clean doesn't remove the cache, so using `local_files_only` here returns the cache even when the local dir is empty.
-    print(f"Checking for {model_name}")
+    logger.info(f"Checking for {model_name}")
```

Comment:
in the future we might wanna move more towards structured logging with some of these

that way we can e.g. group logs by model in DD

`logger.info("Checking for model", extra=dict(model=model_name)"`

def not a right now thing, but worth considering going forward

Reply:
Definitely agree!